test/openshift/e2e/parallel/1-031_validate_toolchain/01-check.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      set -e
+      set -o pipefail
+
+      # These variables need to be maintained according to the component matrix: https://spaces.redhat.com/display/GITOPS/GitOps+Component+Matrix
+      expected_kustomizeVersion='v5.0.1'
+      expected_helmVersion='v3.11.2'
+      expected_argocdVersion='v2.7.6'
+      expected_dexVersion='v2.35.1'
+      expected_redisVersion='6'
```

Comment:
Hi @anandrkskd 
Could you also please add [this](https://gitlab.cee.redhat.com/gitops/operator-e2e/-/commit/3b9ac9fb6f1583edd5b9ef300cf2e40f0e347f89) change?
Thanks